privileges-grantor/README.md

```diff
@@ -11,12 +11,11 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | instance\_name | Privileges Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | lambda\_timeout | The time after which the lambda execution stops. | string | `"200"` | no |
 | memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
-| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `<<EOD "CREATE_TABLE","ALTER_TABLE" EOD` | string | `"\"CREATE_TABLE\",\"ALTER_TABLE\""` | no |
-| database\_filter | List of database names to be added to SNS filter. Supported format: `<<EOD \"DB_NAME_1\",\"DB_NAME_2\" EOD` | string | n/a | yes |
+| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `[ "CREATE_TABLE","ALTER_TABLE" ]` | list | `[ "CREATE_TABLE", "ALTER_TABLE" ]` | no |
+| database\_filter | List of database names to be added to SNS filter. Supported format: `[ "DB_NAME_1", "DB_NAME_2" ]` | list | n/a | yes |
 | metastore\_events\_sns\_topic | SNS Topic for Hive Metastore events. | string | n/a | yes |
-| pg\_jars\_s3\_key | S3 key where zip file is located. | string | n/a | yes |
-| pg\_lambda\_bucket | Bucket where the Lambda zip can be found, for example 'bucket_name'. Used together with `pg_jars_s3_key`. | string | n/a | yes |
-| pg\_lambda\_version | Version of the Privileges Grantor Lambda. | string | n/a | yes |
```

Comment:
Why has this been removed? If on purpose this needs to be documented.

Reply:
I've added a line in the Changelog. Reason being is that it is duplicate to the functionality provided by `pg_jars_s3_key`. 

`pg_jars_s3_key` allows you to specify the path to the lambda zip on S3 (the full path). 

I removed `pg_lambda_version` variable as it was confusion why we need it if we have `pg_jars_s3_key`. However I added an example in the README.md of how when invoking the module the end user can define this variable - `pg_lambda_version` locally and use a null_resource to pull the lambda from maven and upload it to s3 via aws_s3_object resource. then this `aws_s3_object` resource path can be passed to `pg_jars_s3_key` var passing the path to the lambda zip.

Reply:
I would recommend against this nullResource as that means downloading to laptop (or wherever you apply TF) and then uploading to s3. I've had this in a different project and have had so many failures on this and bad TF state afterwards it also means your TF apply command will be super slow. Use the hcom maven repo on s3 and point to that by default. I can point you to some examples how we do this now which works a lot smoother.